docs/sources/operators-guide/deploying-grafana-mimir/jsonnet/memberlist.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Migrating from Consul to Memberlist KV store for hash rings"
+menuTitle: "Migrating from Consul to Memberlist"
+description: "Learn how to migrate from using Consul as KV store for hash rings to using memberlist without any downtime."
+weight: 30
+---
+
+# Migrating from Consul to Memberlist as KV store for hash rings without downtime
```

Comment:
As per docs squad guidelines (as I understood it), this is the expected to match the `title` in the metadata above.

@jdbaldry Is my understanding of the guidelines correct? If so, we could enforce it in the `doc-validator`.

Reply:
I also understand this to be the case. I think we absolutely could add it to the `doc-validator`. The only wrinkle in doing this quickly is that we currently do not parse the front matter as it is not supported by the current markdown parser https://github.com/grafana/technical-documentation/blob/c34c993f9516f2aadb0ad816b74b25a045f59d73/tools/doc-validator/main.go#L192-L193

Reply:
Right. For this specific case, I think we could manually parse the `title` in the frontmatter with a regex. If we'll need to parse more structured fields in the frontmatter, then we could consider to switch to a different library (even if the markdown parser I've used was very handy...).

Reply:
For now I've modified `title: ` and this heading to match.